JobConfig/cosmic/NominalNoFieldCRY.fcl

```diff
@@ -0,0 +1,5 @@
+#include "Production/JobConfig/cosmic/NoField.fcl"
+source.module_type: EmptyEvent
+physics.producers.generate : @local::Cosmic.generateCRY
+outputs.PrimaryOutput.fileName : "dts.owner.CosmicCORSIKANominalNoField.version.sequencer.art"
```

Comment:
The output should be CRY and not CORSIKA here

Reply:
thanks, fixed.